manager/notifier.go

```diff
@@ -544,6 +545,77 @@ func (n *notifier) sendEmailNotification(to string, op notificationOp, a *Alert)
 	return writeEmailBody(wc, *smtpSender, to, status, a, n.alertmanagerURL)
 }
 
+// opGenieMessageCreate is the request for sending a opsGenie notification. We are not specifying all the
+// fields. The Details field is populated with the labels from the alert.
+type opsGenieMessageCreate struct {
+	Message     string            `json:"message"`
+	Description string            `json:"description"`
+	Alias       string            `json:"alias"`
+	Teams       []string          `json:"teams"`
+	Details     map[string]string `json:"details"`
```

Comment:
You might also want to include the `source` field and set that to `n.alertmanagerURL` like we do for PagerDuty notifications?